include/swift/DependencyScan/SerializedModuleDependencyCacheFormat.h

```diff
@@ -193,7 +195,8 @@ using ClangModuleDetailsLayout =
                    FlagIDArrayIDField,        // capturedPCMArgs
                    IdentifierIDField,         // CASFileSystemRootID
                    IdentifierIDField,         // clangIncludeTreeRoot
-                   IdentifierIDField          // moduleCacheKey
+                   IdentifierIDField,         // moduleCacheKey
+                   IsSystemField              // isSystem
```

Comment:
Not my area of expertise 😀 but does this require bumping `MODULE_DEPENDENCY_CACHE_FORMAT_VERSION_MAJOR`?

Reply:
Good catch. Fixed, thank you.